src/__tests__/mailtrap_client.test.ts

```diff
@@ -0,0 +1,114 @@
+import { describe, expect, it, beforeAll, afterEach } from "@jest/globals";
```

Comment:
nice! thanks for setting tests up

Reply:
Not sure which one is "kosher" way:

* `yarn add --dev @jest/globals`
* `yarn add --dev @types/jest`

https://jestjs.io/docs/getting-started#using-typescript